src/main/resources/application.properties

```diff
@@ -7,3 +7,9 @@ spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialec
 ##create, create-drop,validate,update
 spring.jpa.hibernate.ddl-auto = update
 
+spring.mail.host = smtp.mailtrap.io
+spring.mail.port = 2525
+spring.mail.properties.mail.smtp.ssl.enable = false
+spring.mail.username = 16e7fd06e16048
+spring.mail.password = fb871f7072621c
```

Comment:
if it's kindly change it to a random number. but get in touch with me before you change anything